### DISCUSSION TITLE
Klaviyo form embed

### DISCUSSION THREAD

**Discussion by rynfindlay** [breezefront/community#279] [1. Help]
Hi all,

Has anyone successfully embedded a Klaviyo newsletter sign up form?

I have the script rendering successfully on refresh but can't reinitialise the script using breeze:mount when navigating to other pages on the site (the form doesn't show unless I refresh).

I have installed https://github.com/breezefront/module-breeze-klaviyo-reclaim

However, it doesn't seem to take care of the initialization script below:

<img width="990" alt="Screenshot 2024-05-24 at 14 01 37" src="https://github.com/breezefront/community/assets/151278823/38a5fd03-5f08-4c10-ac34-b15207692997">

I have tried using:

`    document.addEventListener('breeze:mount', function () {
        $.getScript('https://static.klaviyo.com/onsite/js/klaviyo.js?company_id=ID_HERE');
    });`

I have also tried refreshing the embed code: <div class="ID_HERE"></div>

Thanks

**Comment by vovayatsyuk** [comment]
Can you please add an instruction on how to test this feature? I'm not familiar with Klaviyo and don't know how to test this case.
